src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,22 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, some street123, #12-12, 11239";
+    private static final String TO_STRING_FORMAT = "%s, %s, %s, %s";
+    private static final String MESSAGE_ADDRESS_CONSTRAINTS =
+            "Person addresses should be: <Block>, <Street>, <Unit>, <Postal Code>";
+    private static final String ADDRESS_VALIDATION_REGEX = "[\\w ]+, [\\w ]+, [\\w \\-#]+, [0-9]+";
```

Comment:
Remember to update documentation (user guide, help, etc) when modifying behavior.